.storybook/manager.ts

```diff
@@ -0,0 +1,12 @@
+import { addons } from '@storybook/manager-api';
+import { create } from '@storybook/theming';
+
+addons.setConfig({
+    theme: create({ 
+        base: "dark", 
+        brandTitle: "Life",     
```

Comment:
What is this ?

Reply:
just some tests I'd forgotten to modify, resolved by this [commit](https://github.com/Popoyoko/popoyoko-ui/pull/201/commits/54cca4a6ba0a5df3bb7e1c3537ea84c506a80b94).